src/components/Tokens/TokenDetails/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { SwapAction } from './types'
+
+/**
+ * Displays the time as a human-readable string.
+ *
+ * @param {number} timestamp - Transaction timestamp in milliseconds.
+ * @param {number} locale - BCP 47 language tag (e.g. en-US).
+ * @returns {string} Message to display.
+ */
+export function getLocaleTimeString(timestamp: number, locale = 'en-US') {
```

Comment:
I believe we already have a function that handles this. See [getTimestampFormatter](https://github.com/Uniswap/interface/blob/main/src/components/Charts/PriceChart/format.ts)

Reply:
I tried these and found that they didn't match up with the display styles in Figma. I can ask design if they prefer any of the existing formats, but otherwise a new formatter is needed. What is your preferred date string representation?

Reply:
You can define your own format as well, ask @JackShort for how to implement